Automatic Validation ended with:
> 2024-02-01T18:31:18.9527121Z ##[error] One or more errors occurred. (Manifest Error: No Suitable Minimum Version:  [PackageIdentifier] Value: Microsoft.PowerToys

(Automated response - build 695.)

--- a/manifests/l/lin-ycv/EverythingPowerToys/0.78.0/lin-ycv.EverythingPowerToys.installer.yaml
+++ b/manifests/l/lin-ycv/EverythingPowerToys/0.78.0/lin-ycv.EverythingPowerToys.installer.yaml
@@ -14,7 +14,6 @@ Dependencies:
     MinimumVersion: 0.78.0
 Installers:
 - Architecture: x64
-  InstallerType: exe
   InstallerUrl: https://github.com/lin-ycv/EverythingPowerToys/releases/download/v0.78.0/EverythingPT-0.78.0-x64.exe
   InstallerSha256: 3e28fa80237a35ff5b5f86acab6b34d4aff34f4af21d8bd59d202f10a0c3c41c
 ManifestType: installer

